sdk/identity/identity/samples/AzureIdentityExamples.md

```diff
@@ -0,0 +1,347 @@
+# Table of contents
```

Comment:
We don't have anything that shows how to do auth in browser apps... :(

Reply:
We have both `Authenticating a user account interactively in the browser` and `Authenticating a user account with auth code flow` - do you think that should be more prominently displayed somehow?

Reply:
I wonder if that should be the very first thing so that we get that out of the way? Maybe prefix the header with `Browser`?

Also, neither of those tell me that they are for browser applications. I can still authenticate interactively in the browser even though I am using a node application....